lib/preboots/setup.js

```diff
@@ -2,29 +2,38 @@
 
 var wrhs = require('warehouse-models');
 var Feedsme = require('../feedsme');
-var Datastar = require('datastar');
-
+const { DynamoDB } = require('aws-sdk');
+var dynamo = require('dynamodb-x');
+const AwsLiveness = require('aws-liveness');
+const liveness = new AwsLiveness();
 var HttpsAgent = require('https').Agent;
 var HttpAgent = require('http').Agent;
 
-module.exports = function setup(app, options, next) {
+module.exports = function setup(app, options = { dynamo: {} }, next) {
```

Comment:
Not sure how much this is going to save you. if we really need to care, we might need to do something like:
```suggestion
module.exports = function setup(app, options = {}, next) {
  const { dynamo = {} } = options;
```